### ISSUE TITLE
The API is working fine but there is a message error (part of API Error) that stops the rest of my program from running

### ISSUE THREAD

**Issue by machelreid** [ConvertAPI/convertapi-python#7]

**Comment by mayaizart** [comment]
I'll be more specific as I've found where the issue is. 
When some sort of error occurs and an `ApiError` is thrown the following exception is thrown:
``` python
File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/convertapi/exceptions.py", line 14, in __str__
    message = "%s Code: %s. %s" % (self.message, self.code, self.invalid_parameters)     
AttributeError: 'ApiError' object has no attribute 'message'
```
The issue is that `self.message` is never defined in ApiError, therefore `__str__ `fails. Adding `self` to where message is defined in `__init__` will fix the issue.

**Comment by tomasr78** [comment]
My idea is that the problem appears when response does not has an actual response model and API error json model binding fails. It could happen when server return 500 for some unknown reason.

**Comment by mayaizart** [comment]
In my case I realised this issue when we sent a faulty file path. Only after making the change above I could see the actual error that was returned which was code 5007 - Unable to download remote file'.
So that's not a random 500 error.

**Comment by laurynas-convertapi** [comment]
Thanks @mayaizart for the fix PR. Merged and released 1.1.1 version.
